packages/reaction-collections/server/publications/products.js

```diff
@@ -7,12 +7,31 @@
 Meteor.publish("Products", function (productScrollLimit, productFilters) {
   check(productScrollLimit, Match.OneOf(null, undefined, Number));
```

Comment:
We can in the way I did. `audit-arguments-check ` only need that all publication arguments were checked using check(). After we can validate structure of any argument using SS validate()